decentraland/unity-explorer
# fix: DebugRoomsSystem allocations

## What does this PR change?

Optimises calls of the system with `SetVisibilityBinding`

## How to test the changes?

1. Launch the explorer
2. Just a technical improvement, ensure that everything works normally

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
